portal/server/types/view-models/record-correction/utilisation-report-correction-information.ts

```diff
@@ -1,11 +1,14 @@
-import { CurrencyAndAmount } from '@ukef/dtfs2-common';
+import { Currency } from '@ukef/dtfs2-common';
 import { BaseViewModel } from '../base-view-model';
 
 export type UtilisationReportCorrectionInformationViewModel = BaseViewModel & {
   backLinkHref: string;
   feeRecord: {
     exporter: string;
-    reportedFees: CurrencyAndAmount;
+    reportedFees: {
+      currency: Currency;
+      amount: string;
```

Comment:
Perhaps it would be good to reference this as formatted?
```suggestion
      formattedAmount: string;
```

Reply:
Pushed thanks